### DISCUSSION TITLE
How can I get posts from discussions tab of the repo?

### DISCUSSION THREAD

**Discussion by deven298** [gitpython-developers/GitPython#1742] [Q&A] [answered]
I am writing a python script to get the discussion posts on a repo to built a GPT. However, I dont see the support to get the posts from discussions tab of the github repo.

**Comment by Byron** [comment, marked as answer]
`GitPython` allows to interact with `git`, not with GitHub. You probably look [for a GitHub client](https://github.com/PyGithub/PyGithub).

**Comment by deven298** [comment]
Thanks, @Byron!
